src/writing-your-own-application/generate-project/esp-template.md

```diff
@@ -39,10 +39,10 @@ Before going further, let's see what these files are for.
 - [.gitignore][gitignore]
     - Tells `git` which folders and files to ignore
 - [Cargo.toml][cargo-toml]
-    - The usual Cargo manifest declaring some meta-data and dependencies of the project
+    - The usual Cargo manifest declares some meta-data and dependencies of the project
```

Comment:
I suggest that you keep "declaring" here.

I think Grammarly treated this as a sentence and proposed an update. However, these are descriptions of files that are mostly written as sentence fragments and noun phrases.

So this edit is not necessary, but good in terms of following the same style with surrounding text.